src/Element/FieldPane.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace Drupal\islandora_form_fieldpanel\Element;
+
+use Drupal\Core\Render\Element;
+use Drupal\Core\Render\Element\FormElement;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Allows for theming & processing fieldpanes.
+ *
+ * @FormElement("fieldpane")
+ */
+class FieldPane extends FormElement {
+
+  /**
+   * Constants.
+   */
+  const DELETEBUTTON = 'delete-fieldpane';
+  const MOVEUPBUTTON = 'move-up-fieldpane';
+  const MOVEDOWNBUTTON = 'move-down-fieldpane';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getInfo() {
+    $info = [
+      '#input' => TRUE,
+      '#process' => ['islandora_form_fieldpanel_fieldpane_process'],
+      '#user_data' => ['add' => TRUE, 'delete' => TRUE],
+      '#theme_wrappers' => ['fieldpane'],
+    ];
+
+    return $info;
+  }
+
+  /**
+   * FieldPane's theme hook.
+   *
+   * @param array $element
+   *   The element.
+   *
+   * @return string
+   *   The processed string.
+   */
+  public static function theme(array $element) {
+    $children = isset($element['#children']) ? $element['#children'] : '';
+    $description = isset($element['#description']) ? "<div class='description'>{$element['#description']}</div>" : '';
+    return "<div id='{$element['#hash']}' class='islandora-form-fieldpanel-pane'>{$description}{$children}</div>";
```

Comment:
Could... errant angle-brackets in the description break the markup?

Reply:
Markup generated is valid; however, looks like `<` will cause the descriptions to be truncated?

Reply:
Cross-referencing: D7 is worse, generating broken markup... so... It's better, I guess? Shrug?